npms/skale-ima/imaEventLogScan.mjs

```diff
@@ -188,6 +127,14 @@ export async function safeGetPastEventsProgressive(
             nBlockFrom, nBlockTo, joFilter
         );
     }
+    if( log.verboseGet() >= log.verboseReversed().information ) {
+        details.write( strLogPrefix +
+            cc.info( "Will run progressive event log search for event " ) +
+            cc.j( strEventName ) + cc.info( " via URL " ) +
+            cc.u( owaspUtils.ethersProviderToUrl( ethersProvider ) ) +
+            generateWhileTransferringLogMessageSuffix( optsChainPair ) +
+            cc.info( "..." ) );
+    }
```

Comment:
What is the purpose of this log? There was no such log in reverted PR

Reply:
This is requested by Evg - print node number we requesting